src/ImageSharp/Formats/Jpg/JpegEncoderCore.cs

```diff
@@ -2,295 +2,169 @@
 // Copyright (c) James Jackson-South and contributors.
 // Licensed under the Apache License, Version 2.0.
 // </copyright>
-
 namespace ImageSharp.Formats
 {
     using System;
+    using System.Buffers;
     using System.IO;
+    using System.Numerics;
+    using System.Runtime.CompilerServices;
+
+    using ImageSharp.Formats.Jpg;
 
     /// <summary>
-    /// Image encoder for writing an image to a stream as a jpeg.
+    ///     Image encoder for writing an image to a stream as a jpeg.
     /// </summary>
     internal unsafe class JpegEncoderCore
     {
         /// <summary>
-        /// The number of quantization tables.
+        ///     The number of quantization tables.
```

Comment:
Yeah, easy enough to fix. Don't worry about it. 

This is by far the best plugin for VS to remove trailing whitespace btw, it does it automatically on save.
https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer